server/src/operators/card_operator.rs

```diff
@@ -545,6 +545,7 @@ pub fn get_metadata_query(
                 user_columns::username,
                 user_columns::website,
                 user_columns::visible_email,
+                user_columns::api_key_hash,
```

Comment:
Do we want to include their api key in the return request? seems like it might be a security issue